### DISCUSSION TITLE
Mapped property is missing

### DISCUSSION THREAD

**Discussion by stanlygoodspeed** [microsoft-search/pnp-modern-search#1904] [Q&A]
**Version used**
4.5.3

**Describe the bug**
We map user properties to a managed property. In Delve, users have filled in fields. The fields are filled in by a group of term sets. Managed properties are mapped to RefinableStringXX. But the selected RefinableString properties in the search results do not give values.

**Screenshots**

![alt tag](https://i.ibb.co/r3MvMnF/mapped.jpg "Mapped property")​

![alt tag](https://i.ibb.co/dr6qhp6/null.jpg "Property is null")​

**Desktop (please complete the following information):**
 - Browser [chrome, firefox, edge, safari]

**Comment by wobba** [comment]
This is a mapping configuration issue, and not a web part issue. See https://github.com/wobba/SPO-Trigger-Reindex for info on people indexing.
